src/tests/integration/test-data/foundry-prove-skip-legacy

```diff
@@ -60,6 +60,7 @@ BytesTypeTest.testFail_bytes32(bytes32)
 BytesTypeTest.testFail_bytes4(bytes4)
 ChainIdTest.test_chainid_setup()
 CoinBaseTest.test_coinbase_setup()
+ConstructorTest.test_constructor()
```

Comment:
Thanks for adding this test! For it to be run on CI (in this case, only with the booster backend), it should also be added to [foundry-prove-all](https://github.com/runtimeverification/kontrol/blob/master/src/tests/integration/test-data/foundry-prove-all).